boot/CMakeLists.txt

```diff
@@ -59,8 +59,8 @@ set(ISO_VOLNAME      "ReactOS")             # For both the Volume ID and the Vol
 
 ## BootCD
 # Create the file list
-file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/bootcd.lst "")
-file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/bootcd.lst "${CMAKE_CURRENT_BINARY_DIR}/empty\n")
+file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/bootcd.cmake.lst "")
+file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/bootcd.cmake.lst "${CMAKE_CURRENT_BINARY_DIR}/empty\n")
```

Comment:
Question: having the ".cmake." appended in the file name means that the file will contain also cmake commands in it?

Reply:
We've talked about this on IRC, but I'll say it again here so it's on the record: the file will contain generator expressions like $<TARGET_FILE:some_module>. So it needs to be processed by CMake, but doesn't contain CMake commands.